CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+# Contributing to opentelemetry-ebpf-profiler
+
+The Go special interest group (SIG) meets regularly. See the
+OpenTelemetry
+[community](https://github.com/open-telemetry/community)
+repo for information on this and other SIGs.
```

Comment:
There are many SIG meetings and over time there will be even more. Maybe [calendar-profiling](https://groups.google.com/a/opentelemetry.io/g/calendar-profiling) or the slack channel [#otel-profiles](https://cloud-native.slack.com/archives/C03J794L0BV) could be mentioned for easier access.